packages/ui-kit/src/components/markdown.tsx

```diff
@@ -255,7 +256,8 @@ const InlineCode = styled.code`
   border: 1px solid ${colors.light.borderSecondary};
   border-radius: ${dimensions.spacings.xs};
   color: ${colors.light.textCode};
-  font-family: ${typography.fontFamilies.code};
+  font-family: ${typography.fontFamilies.nonMonospaceCode};
+  font-feature-settings: 'salt';
```

Comment:
at least my opinion:  apply `ss02` and letter-spacing=0 here vs globally.   But Zoe's view wins if she thinks we should have wider spacing or disambiguation everywhere .

Reply:
I think @zbalek should have a final call on it, because I'm not sure anymore which approach she wants to do here.
Either we have a consistent font configuration for all text components or we have a special one for string literals and other places where it makes sense as Nikolaus suggested.